aminator/plugins/provisioner/apt.py

```diff
@@ -134,9 +148,33 @@ def apt_get_update(self):
     def apt_get_clean():
         return monitor_command(['apt-get', 'clean'])
 
+    @staticmethod
+    def apt_get_install(*options):
+        install_result = monitor_command(['apt-get', '-y', 'install'] + options)
+        if not install_result.success:
+            log.debug('failure:{0.command} :{0.std_err}'.format(install_result.result))
+        return install_result
+
     @classmethod
-    def apt_get_install(cls, package):
-        return monitor_command(['apt-get', '-y', 'install', package])
+    def _install(cls, package):
+        return cls.apt_get_install(package)
+
+    @classmethod
+    def install(cls, package, local_install=False):
```

Comment:
This logic is common to apt and aptitude, we let overridden _install and _localinstall methods do the plugin-specific foo.